Mathlib/LinearAlgebra/DirectSum/Finsupp.lean

```diff
@@ -16,43 +16,31 @@ The tensor product of `ι →₀ M` and `κ →₀ N` is linearly equivalent to
 -/
 
 
-universe u v w
-
 noncomputable section
+suppress_compilation
 
-open DirectSum
-
-open Set LinearMap Submodule
-
-section TensorProduct
+open DirectSum Set LinearMap Submodule TensorProduct
 
-open TensorProduct
-
-open TensorProduct
+variable (R M N ι κ : Type*) [CommSemiring R] [AddCommMonoid M] [Module R M]
+  [AddCommMonoid N] [Module R N]
```

Comment:
A style nit: putting the types on a separate line to the typeclasses makes adding new types to the section slightly less painful, as then you don't have to rewrap everything.

Reply:
Will do